### ISSUE TITLE
[DocDB] flaky test: GeoTransactionsTest.TestTransactionTableDeletion 

### ISSUE THREAD

**Issue by es1024** [yugabyte/yugabyte-db#19403]
Jira Link: [DB-8195](https://yugabyte.atlassian.net/browse/DB-8195)
### Description

The test GeoTransactionsTest.TestTransactionTableDeletion infrequently fails with the following error:

```
../../src/yb/yql/pgwrapper/geo_transactions-test.cc:619
Expected equality of these values:
  1
  count
    Which is: 0
```

### Issue Type

kind/bug

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-8195]: https://yugabyte.atlassian.net/browse/DB-8195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by rthallamko3** [comment]
Stability [report](https://detective.dev.yugabyte.com/stability/test?analyze_trends=true&branch=master&build_type=all&class=GeoTransactionsTest&fail_tag=all&name=TestTransactionTableDeletion&platform=all) looks clean. @es1024 , Is this something that is still relevant?

**Comment by es1024** [comment]
I think this got fixed somewhere between 2.20 (has failures) and 2024.1 (no failures)

**Comment by rthallamko3** [comment]
thanks @es1024 , lets close it and recreate it if needed.
